### DISCUSSION TITLE
Hosting a postgresql db in the cloud

### DISCUSSION THREAD

**Discussion by A-curious-guy** [community/community#135505] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I used to code web apps with the MERN stack, and currently i'm learning postgresql, so i want to know where can i host my postgresql online so that i can deploy the app later, something like mongodb atlas.

**Comment by AghilesTMA** [comment, marked as answer]
Personally I use supabase:https://supabase.com/ which i really like, it provides you with a posgresql database that i use it with with drizzle-orm in my node/ts app.
I have also heard of neon but i didn't try it personally, Neon: https://neon.tech/
